### DISCUSSION TITLE
[packaging] Linux app icon

### DISCUSSION THREAD

**Discussion by xgqt** [cyanfish/naps2#443] [Development]
Sorry for dumb question :D but the Linux desktop file references a certain app icon. Where that icon could be found?

See also: https://bugs.gentoo.org/916780#c3

**Comment by cyanfish** [comment]
Here: https://github.com/cyanfish/naps2/blob/master/NAPS2.Lib/Icons/scanner-128.png

It gets copied in the build process:
https://github.com/cyanfish/naps2/blob/778e7d7569610b006a3e5e175de3e67f13d65fd9/NAPS2.Tools/Project/Packaging/DebPackager.cs#L50

**Reply by xgqt** [reply]
Thanks a lot! Fixed in https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=f04d63b2c834afee524ff8753cc02d3da7eb4d7a
